sass/components/_on-this-page.scss

```diff
@@ -12,29 +11,24 @@
     padding-inline-start: $h-padding;
   }
 
-  >ul {
+  > ul {
     padding-inline: $h-padding;
   }
 
-  li[data-active=true] {
-    font-weight: bolder;
-
-    li {
-      font-weight: normal;
-    }
+  li[data-active="true"] > a {
+    color: #ececec;
   }
```

Comment:
I tried this with bold too, but with white it looked too distracting in my opinion. I can change it back to bold or some in-between weight (if the font supports it) in case others prefer it though

Reply:
I wouldn't use bold as in some rare cases the text might jump. Just using color should be enough.